icons/file_type_doppler.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient id="a" x1="66.248" y1="-2.76" x2="7.594" y2="75.91" gradientUnits="userSpaceOnUse" gradientTransform="matrix(.33735 0 0 .33735 2 2)"><stop offset=".036" stop-color="#409FF8"/><stop offset=".5" stop-color="#1673FF"/><stop offset="1" stop-color="#516FF7"/></linearGradient></defs><path fill-rule="evenodd" clip-rule="evenodd" d="M2.514 2a.282.282 0 00-.235.436l5.02 7.857a2.258 2.258 0 001.902 1.043h6.789c2.489 0 4.045 2.2 4.035 4.664-.009 2.464-1.666 4.642-4.035 4.642H8.741a.751.751 0 00-.751.754v7.852a.75.75 0 00.751.752H18c8.012 0 11.995-7.602 12-14 .006-6.398-3.931-14-12-14H2.514zm3.294 18.664H4.005C2.898 20.664 2 21.708 2 22.997v6.798c0 .111.091.201.203.201h1.802c1.108 0 2.006-1.044 2.006-2.333v-6.797a.202.202 0 00-.203-.202z" fill="url(#a)"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><linearGradient id="linear-gradient" x1="24.19" y1="33.05" x2="4.41" y2="6.51" gradientTransform="matrix(1, 0, 0, -1, 0, 34)" gradientUnits="userSpaceOnUse"><stop offset="0.04" stop-color="#409ff8"/><stop offset="0.5" stop-color="#1673ff"/><stop offset="1" stop-color="#516ff7"/></linearGradient></defs><title>file_type_doppler</title><path d="M2.51,2a.28.28,0,0,0-.23.44l5,7.85a2.25,2.25,0,0,0,1.9,1.05H16c2.49,0,4,2.2,4,4.66s-1.67,4.64-4,4.64H8.74A.75.75,0,0,0,8,21.4v7.85a.75.75,0,0,0,.75.75H18c8,0,12-7.6,12-14S26.07,2,18,2Zm3.3,18.66H4A2.19,2.19,0,0,0,2,23v6.79a.2.2,0,0,0,.2.21H4a2.2,2.2,0,0,0,2-2.34V20.87A.2.2,0,0,0,5.81,20.66Z" style="fill-rule:evenodd;fill:url(#linear-gradient)"/></svg>
```

Better SVG that loads in Illustrator, taken from the logo in https://www.doppler.com/

Reply:
Thanks for this!